tests/test_metrics.py

```diff
@@ -0,0 +1,57 @@
+from unittest import mock
+
+import jax.numpy as jnp
+import pytest
+from jax import random
+
+from blackjax.inference import metrics
+
+KEY = random.PRNGKey(0)
+
+
+@pytest.fixture()
+def patch_jax(monkeypatch):
```

Comment:
this is (`monkeypatch`) used here?

Reply:
Yes its monkeypatch? I feel like I'm missing a question hee

Reply:
sorry, ignore me, you have `monkeypatch.setattr(metrics.jnp, "multiply", multiply)`, not sure what I was thinking :flushed:

Reply:
No worries! I appreciate the review :)